src/cli/init_command.zig

```diff
@@ -41,6 +41,9 @@ pub const InitCommand = struct {
 
         const input = try std.io.getStdIn().reader().readUntilDelimiterAlloc(alloc, '\n', 1024);
         if (input.len > 0) {
+            if (strings.endsWithChar(input, '\r')) {
```

Comment:
what if the only character is a `\r`?

Reply:
then it will be incorrectly an empty slice. will fix